src/components/page-active-job-component/style.css

```
@@ -1,6 +1,7 @@
-.scheduled-job {
-  display: grid;
-  grid-auto-flow: column;
-  grid-auto-columns: max-content;
-  gap: var(--eds_spacing_medium);
+.scheduled-job__link {
+  text-decoration: none;
+}
+.scheduled-job__link:hover {
+  color: var(--eds_interactive_primary__hover);
```

this is not the same color as the link, which is using --eds_interactive_primary__resting
```suggestion
  color: var(--eds_interactive_primary__resting);
```

---

fixed